packages/app/src/templates/actions/update-template-order.action.ts

```diff
@@ -9,12 +9,15 @@ import { UpdateTemplateOrderError } from "App/templates/errors"
 import { updateTemplateOrderRequest } from "App/templates/requests"
 import { Template } from "App/templates/dto"
 
-export const updateTemplateOrder = createAsyncThunk<Error | Template, Template>(
+export const updateTemplateOrder = createAsyncThunk<
```

Comment:
How about showing failure/success messages/popups, as shown in designs/ACC? Would it be a scope of upcoming PR's? 

https://user-images.githubusercontent.com/41268731/177121711-1ac461a3-8810-4ffe-955b-a2b0f8461442.mov

Reply:
I will add them in separate PR